src/main/java/com/elastic/support/util/JsonYamlUtils.java

```diff
@@ -181,21 +122,11 @@ public static void buildFlattenedMap(Map result, Map source, String path) {
       }
    }
 
-   private static Map nullSafeYamlMap(Map doc){
+   private static Map<String, Object> nullSafeYamlMap(Map<String, Object> doc){
       if (doc == null){
-         doc = new HashMap();
-      }
-      return doc;
-   }
-
-   private static Map listToMap(List input){
-      int sz = input.size();
-      Map output = new LinkedHashMap<>();
-      for(int i=0; i < sz; i++){
-         output.put("idx_" + i, input.get(i));
+         doc = new HashMap<>();
```

Comment:
Nitpick: no reason to touch the reference.

```suggestion
         return new HashMap<>();
```

Reply:
Done and also removed some other unused code in https://github.com/elastic/support-diagnostics/pull/487/commits/279345dff9ef69ad6cbd481fae771f7f8a210743.